ui/imports/shared/popups/send/views/AmountToSendNew.qml

```diff
@@ -140,9 +140,9 @@ Control {
                 return SQUtils.AmountsArithmetic.times(
                             SQUtils.AmountsArithmetic.fromString(inputDelocalized),
                             SQUtils.AmountsArithmetic.fromNumber(
-                                price * (10 ** root.fiatDecimalPlaces))).round().toFixed()
```

Comment:
The `round()` shouldn't happen here, otherwise we display stuff like "0.00 USD" instead of the correct "<0.01 USD"

Reply:
One of the SwapModal tests caught this